easybuild/easyconfigs/n/num2words/num2words-0.5.10-foss-2021a.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = "PythonBundle"
+
+name = 'num2words'
+version = '0.5.10'
+
+homepage = "https://github.com/savoirfairelinux/num2words"
+description = "Modules to convert numbers to words. 42 --> forty-two"
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+dependencies = [
+    ('Python', '3.9.5'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    ('docopt', '0.6.2', {
```

Comment:
`docopt` 0.6.2 is already included with `Python` 3.9.5, so there's no need to install it again?

That also implies that just using `PythonPackage` as easyblock makes more sense